app/_components/PoiPhotoToggle.tsx

```diff
@@ -0,0 +1,91 @@
+import { useState, useEffect } from "react";
+import { Button } from "./ui/button";
+import { Pin } from "../_utils/global";
+import {
+  Coordinates,
+  GetDistanceFromCoordinatesToMeters,
+} from "../_utils/coordinateMath";
+
```

Comment:
Import Image from NEXT to use for image components
```suggestion
import Image from "next/image";
```